mediathread/templates/main/instructor_dashboard.html

```diff
@@ -11,7 +11,8 @@
 
```

Comment:
The template incorrectly overrides the coursetitle block. Can we keep the look/feel of this page consistent with the rest of the dashboard pages until Z gets a chance to work her magic?